zone.js

```diff
@@ -114,38 +114,25 @@ Zone.patchSetClearFn = function (obj, fnNames) {
   }).
   forEach(function (name) {
     var setName = 'set' + name;
-    var clearName = 'clear' + name;
     var delegate = obj[setName];
 
     if (delegate) {
+      var clearName = 'clear' + name;
       var ids = {};
 
-      if (setName === 'setInterval') {
-        zone[setName] = function (fn) {
-          var id;
-          arguments[0] = function () {
-            delete ids[id];
```

Comment:
Should we really `delete ids[id];` here ?

By deleting, `clearInterval()` will not call the `dequeueTask()` hook if called after the callback as triggered at least once, not sure if it makes sense ?